app/models/section.rb

```diff
@@ -20,4 +21,16 @@ def conflicts_with(section)
     end
     false
   end
+
+  def sort_periods
+    periods_info = periods_day.zip(periods_start, periods_end, periods_type)
+    periods_info = periods_info.sort!.transpose
+    self.periods_day, self.periods_start, self.periods_end, self.periods_type = periods_info
+  end
+
+  def period_info_changed?
+    if(self.changed & ["periods_start", "periods_end", "periods_day", "periods_type"]).any?
+      return true
```

Comment:
Control flow is not needed, and `return` should be omitted. This function can be reduced to one line by removing the if()...end.

i.e., 
`(self.changed & ["periods_start", "periods_end", "periods_day", "periods_type"]).any?`